kong/db/strategies/postgres/init.lua

```diff
@@ -483,22 +479,60 @@ local function execute(strategy, statement_name, attributes, is_update)
 
   clear_tab(argv)
 
-  for i = 1, argc do
-    local name  = argn[i]
-    local value
+  local update = options and options.update
```

Comment:
my 2cts: `is_update` (which you previously came up with) was more readable :sweat_smile: